views/profiles/show.ejs

```diff
@@ -3,10 +3,119 @@
 <div class="container profile">
   <div class="jumbotron">
     <h2><%= profile.name %></h2>
+    <h2 id=profileId hidden="true"><%= profile.id %></h2>
     <img src="<%= profile.photo %>">
   </div>
+ <div>
+  <% if (user) { %>
+ <div class="container">
+   <form>
+     <input id="sendEmail" type="email" required="true">
+     <button id="sendBtn">Send Details</button>
+        <div id="email-sent-success" class="alert alert-success alert-dismissable" hidden="true">
+            <button type="button" class="close" data-dismiss="alert" aria-hidden="true">&times;</button>
+            Success! Profile details sent to 
+        </div>
+        <div id="email-sent-failed" class="alert alert-failure alert-dismissable" hidden="true">
+            <button type="button" class="close" data-dismiss="alert" aria-hidden="true">&times;</button>
+            Failure! Profile details not sent to 
+        </div>
+  </form>
+    <div id="response"></div>
+ </div>
+  <% } %>
+  <div>
+    <table class="table table-user-information">
+      <tbody>
+         <% if (profile.triggeredOn) { %>
+        <tr>
+          <td>Date Triggered:</td>
+          <td><%= profile.triggeredOn %></td>
+        </tr>
+        <% } %>
+        <% if (profile.birthday) { %>
+        <tr>
+          <td>Date of Birth:</td>
+          <td><%= profile.birthday %></td>
+        </tr>
+        <% } %>
+        <% if (profile.gender) { %>
+        <tr>
+          <td>Gender:</td>
+          <td><%= profile.gender %></td>
+        </tr>
+        <% } %>
+        <% if (profile.email) { %>
+        <tr>
+          <td>Email</td>
+          <td><a href="mailto:<%= profile.email %>"><%= profile.email %></a></td>
+        </tr>
+        <% } %>
+        <% if (profile.phonenumber) { %>
+        <tr>
+          <td>Phone Number</td>
+          <td><%= profile.email %></td>
+        </tr>
+         <% } %>
+        <% if (profile.social) { %>
+        <tr>
+          <td>Social</td>
+          <td>
+          <% if (profile.social.twitter) { %>
+            <a href="https://twitter.com/<%= profile.social.twitter %>" class="fa fa-twitter">Twitter</a><br>
+          <% } %>
+           <% if (profile.social.instagram) { %>
+            <a href="https://instagram.com/<%= profile.social.instagram %>" class="fa fa-instagram">Instagram</a><br>
+          <% } %>
+          <% if (profile.social.facebook) { %>
+            <a href="https://facebook.com/<%= profile.social.facebook %>" class="fa fa-facebook">Facebook</a><br>
+          <% } %>
+          </td>
+        </tr>
+        <% } %>
+      </tbody>
+    </table> 
+  </div>
 </div>
 <script>
+  $("#sendBtn").click(function(){
+    var id = document.getElementById("profileId").value
```

Comment:
We don't really need to put the profile id in a hidden H2 element. You could for example simply set `var id = "<%= profile.id %>";` however if you wish to keep this method, then I would suggest placing it in a hidden text field in a form vs a H2 element.